### PR TITLE
DEVDOCS-2832: Abandoned Cart Developer implementation guidelines

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2832](https://jira.bigcommerce.com/browse/DEVDOCS-2832)

## What changed?
* added first version of the article

### CODE REVIEW

File: docs/api-docs/building-storefronts/abandoned-carts.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+  "matching": "*",
+  "route": "/my-abandoned-cart-page/",
+  "type": "recover_abandoned_cart"
+}
+
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/sites/site-routes/post-site-route#requestrunner)
+
+3. To create a customer, send a `POST` request to the `/customers` endpoint. In the request body, set `accepts_product_review_abandoned_cart_emails` to `true` to enable Abandoned Cart Saver notifications. This will create a customer account optimized to receive Abandoned Cart Saver emails.
```
Comment (markcmurphy) [inline docs/api-docs/building-storefronts/abandoned-carts.md:205]: The item would also apply to stores that are using embedded or one-page checkout, correct? If the default value of `accepts_product_review_abandoned_cart_emails` is `false` for customers created through the API (and it seems to be), A developer would need to make sure this is set for any customer that isn't being created as part of the post-transaction embedded checkout flow where it respects UI settings.

File: docs/api-docs/building-storefronts/abandoned-carts.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+The [Abandoned Cart Saver](https://support.bigcommerce.com/s/article/Using-the-Abandoned-Cart-Saver?language=en_US) is a built-in tool that sends automated drip emails to customers who have abandoned their carts. These cart recovery emails contain a link that routes shoppers back to the cart page where they can complete the order.
+
+![Complete your purchase email](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Abandoned%20Carts/01-abandoned-carts.png "Complete your purchase email")
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+> A cart is considered abandoned after a customer leaves the cart without attempting payment and the cart has been inactive for one hour.
```
[the remark this answers is not in the document]
Comment (ektravel) [reply docs/api-docs/building-storefronts/abandoned-carts.md:29]: That's correct. You can customize the email cadence; however, for a cart to be considered abandoned it has to be inactive for one hour.

File: docs/api-docs/building-storefronts/abandoned-carts.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
+# Creating Abandoned Cart Experience
```
[the remark this answers is not in the document]
Comment (ektravel) [reply docs/api-docs/building-storefronts/abandoned-carts.md:1]: Great point! I think "Recovering Abandoned Carts" is more comprehensive.

**Comment by ektravel** [comment]
@uwikaiddi this PR is ready for final review.

File: docs/api-docs/building-storefronts/abandoned-carts.md [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+    "created_at": "2021-08-23T20:02:11Z",
+    "id": 1001,
+    "updated_at": "2021-08-23T20:02:11Z",
+    "url": "http://commerce-zr8y-teststore-bigcommerce.vercel.app"
+  },
+  "meta": {}
+}
+```
+
+You can locate your channel ID by:
```
Comment (markcmurphy) [inline docs/api-docs/building-storefronts/abandoned-carts.md:67]: There is something that it doesn't like about the page breaks so GitHub isn't rendering the MD correctly. The following seemed to work, but you might need to play around with it: 

```md
You can locate your channel ID by:

- going to **Channel Manager > Storefronts** in the control panel, clicking on **...** next to your headless storefront, and selecting **Advanced settings** from the dropdown

  **or**

- sending a `GET` request to the [Get All Channels](https://developer.bigcommerce.com/api-reference/store-management/channels/channels/listchannels) endpoint. 
```
You can locate your channel ID by:

- going to **Channel Manager > Storefronts** in the control panel, clicking on **...** next to your headless storefront, and selecting **Advanced settings** from the dropdown

  **or**

- sending a `GET` request to the [Get All Channels](https://developer.bigcommerce.com/api-reference/store-management/channels/channels/listchannels) endpoint.

**Review by markcmurphy — APPROVED** [review]

File: docs/api-docs/building-storefronts/abandoned-carts.md (hunk 4 above)
Comment (ektravel) [reply docs/api-docs/building-storefronts/abandoned-carts.md:67]: I removed the page breaks completely.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**